### PR TITLE
Update dcos-diagnostics

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

Mark dead services as unhealthy 
Refs: https://github.com/dcos/dcos-diagnostics/pull/202
Fixes: https://jira.d2iq.com/browse/COPS-6514

## Corresponding DC/OS tickets (required)

  - [COPS-6514](https://jira.d2iq.com/browse/COPS-6514) 

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @pierrebeitz  @DominikDary

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy @pierrebeitz @DominikDary @vinodkone

File: CHANGES.md [hunk 1]
```diff
@@ -53,3 +53,5 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Marathon apps have support for CSI volumes. (MARATHON-8765)
 
 * Stop all services at once during upgrade. (COPS-6512)
+
+* Mark inactive service as unhealthy. (COPS-6514)
```
Comment (jongiddy) [inline CHANGES.md:57]: This is only the Telegraf service that is now marked unhealthy if inactive? The changelog should name the specific service. Do we need to do the same for other services?

Comment (janisz) [reply CHANGES.md:57]: all inactive services will be marked as unhealthy

**Comment by janisz** [comment]
@mesosphere-mergebot test all

**Comment by DominikDary** [comment]
This looks stale @janisz , do we want to merge this before the code freeze today?

**Comment by janisz** [comment]
This is realted to COPS-6514 but I think we cannot simply fix this so I'll close this PR

**Pull request closed without merging**